src/Examples/UnitModels/Operations/heat_exchanger_lc_testing.ipynb

```diff
@@ -0,0 +1,315 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Heat Exchanger Lumped-Capacitance Unit Model with Two Property Packages\n",
+    "\n",
+    "![](heat_exchanger_4.svg)\n",
+    "\n",
+    "**Problem Statement**: In this example, we will be heating a benzene-toluene mixture using steam. \n",
+    "\n",
+    "**Tube Side Inlet**\n",
+    "\n",
+    "Flow Rate = 250 mol/s\n",
+    "\n",
+    "Mole fraction (Benzene) = 0.4\n",
+    "\n",
+    "Mole fraction (Toluene) = 0.6\n",
+    "\n",
+    "Pressure = 101325 Pa\n",
+    "\n",
+    "Temperature = 380 K\n",
+    "\n",
+    "**Shell Side Inlet**\n",
+    "\n",
+    "Flow Rate = 100 mol/s\n",
+    "\n",
+    "Mole fraction (Steam) = 1\n",
+    "\n",
+    "Pressure = 101325 Pa\n",
+    "\n",
+    "Temperature = 400 K\n",
+    "\n",
+    "The 0D lumped-capacitance heat exchanger unit model makes the assumption that the temperature difference bewtween discrete \"lumps\" in the heat exchanger is negligible. Likewise, this model can be useful for transient simulations in which temperature gradients within the heat exchanger can be ignored. This example will demonstrate the simulation of the lumped-capacitance heat exchanger by fixing the following degrees of freedom:\n",
+    "- heat transfer area\n",
+    "- overall heat transfer coefficients (one for cold-side and one for hot-side)\n",
+    "\n",
+    "\n",
+    "\n",
+    "IDAES documentation reference for heat exchanger 0D model: https://idaes-pse.readthedocs.io/en/latest/reference_guides/model_libraries/generic/unit_models/heat_exchanger.html\n",
+    "\n",
+    "The IDAES library contains a more advanced `HeatExchangerLumpedCapacitance` model supporting a wall temperature and heat holdup for transient simulations; more details on the lumped capacitance heat exchanger may be found [here](https://idaes-pse.readthedocs.io/en/latest/reference_guides/model_libraries/generic/unit_models/heat_exchanger_lc.html)."
```

Comment:
This line is a holdover from one of the other notebooks, however this example actually is the HXLC and I would suggest rephrasing these lines to emphasize the LC documentation first and then reference the simple 0D model.